Source/Core/Core/PowerPC/JitArm64/JitArm64_SystemRegisters.cpp

```diff
@@ -622,30 +622,31 @@ void JitArm64::mfcr(UGeckoInstruction inst)
     ARM64Reg CR = gpr.CR(i);
     ARM64Reg WCR = EncodeRegTo32(CR);
 
-    // SO
+    // SO and LT
+    static_assert(PowerPC::CR_SO_BIT == 0);
+    static_assert(PowerPC::CR_LT_BIT == 3);
+    static_assert(PowerPC::CR_EMU_LT_BIT - PowerPC::CR_EMU_SO_BIT == 3);
     if (i == 0)
     {
-      UBFX(XA, CR, PowerPC::CR_EMU_SO_BIT, 1);
+      UBFX(XA, CR, PowerPC::CR_EMU_SO_BIT, 4);
+      ANDI2R(WA, WA, ~(PowerPC::CR_EQ_BIT | PowerPC::CR_GT_BIT));
```

Comment:
I think you used the wrong enum values. The `_BIT` ones are the shift amounts. You want the other ones that are masks. (The naming is confusing imo, feel free to rename, including the ones I added.)

Reply:
Yep. Fixed.